package.json

```diff
@@ -89,5 +89,9 @@
   },
   "engines": {
     "node": ">=20.18.1"
+  },
+  "packageManager": "yarn@1.22.17",
+  "resolutions": {
+    "@kiltprotocol/type-definitions": "1.11501.0-rc.1"
   }
```

Comment:
the sdk rc doesn't use the rc type definitions by default so I had to force this here